src/types.ts

```diff
@@ -36,6 +36,7 @@ export enum AutarkyType {
 
 export interface sunsynkPowerFlowCardConfig extends LovelaceCardConfig {
     type: string;
+    dynamicConfig: boolean;
```

Comment:
Is this needed?

Reply:
Definitely not. Wanted to add the functionality for the dynamic hiding of elements, but just seems to much maintenance for too little gain at this stage.